environment.yml

```diff
@@ -4,24 +4,7 @@ channels:
  - conda-forge
 dependencies:
  - python==3.8
- - numpy>=1.21.2
- - pandas>=1.3.3
- - scipy>=1.0.0
- - anndata==0.7.8
- - zarr>=2.5.0
- - numcodecs>=0.7.2
- - nbsphinx==0.8.5
- - pandoc>=2.14.2
- - pytest>=6.2.4
  - jupyterlab==3.1.14
- - jupyter_server==1.11.0
- - traitlets==4.3.3
- - markupsafe==2.0.1
```

Comment:
Not sure where `traitlets` or `markupsafe` are used in this project. `traitlets` is a dependency of `ipywidgets`, which is installed as a part of `vitessce`.

Reply:
Markupsafe broke lots of things in a minor release as referenced in this issue https://github.com/pallets/markupsafe/issues/284

Reply:
Ok! Do you have a suggestion of what "extras_require" these dependencies should be pinned for? I'm guessing testing? But If they are necessary for `vitessce` to work, then we should probably add to `install_requires` (as dependencies for this package) instead.